src/actions/ActionsMakeshift.ts

```diff
@@ -0,0 +1,86 @@
+import { Target } from "../typings/target.types";
+import sleep from "../util/sleep";
+import CommandAction from "./CommandAction";
+
+abstract class MakeshiftAction extends CommandAction {
+  abstract command: string;
```

Comment:
Wow TIL.  Didn't realise you could declare non-function properties abstract 😄

Reply:
Cool isn't it? :D